src/WebJobs.Script.WebHost/Security/KeyManagement/DefaultSecretManagerProvider.cs

```diff
@@ -39,7 +38,7 @@ public sealed class DefaultSecretManagerProvider : ISecretManagerProvider
             _environment = environment ?? throw new ArgumentNullException(nameof(environment));
             _hostNameProvider = hostNameProvider ?? throw new ArgumentNullException(nameof(hostNameProvider));
 
-            _logger = loggerFactory.CreateLogger(ScriptConstants.LogCategoryHostGeneral);
+            _loggerFactory = loggerFactory;
```

Comment:
We want all secrets log under `LogCategoryHostGeneral` category. Did we decide to use full type name instead?

Reply:
Yes I changed this because of feedback from Brett, he said we were moving away from log categories and wanted to use type names